core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/PpwStorageTransferJob.scala

```diff
@@ -67,6 +65,18 @@ object PpwStorageTransferJob {
       message
     )
   }
+
+  def forTesting: PpwStorageTransferJob = PpwStorageTransferJob(
```

Comment:
Could we put this in a test file instead of a main file if it is for testing?

Reply:
Sure!